### PR TITLE
Changes suggested on devdesk

### PR DESCRIPTION
**Pull request by tahmeedshah**
Changes suggested on devdesk-419 and DEVDESK-427

### CODE REVIEW

**Review by blaisefonseca — APPROVED** [review]

File: app/views/forms/view/view-contact.directive.html [hunk 1]
```diff
@@ -78,15 +78,15 @@ <h4>
 			<address class="km-value">
 				<div class="km-pre">{{document.address | lstring:locale}}</div>
 				{{document.city | lstring:locale}}
-				<span ng-show="document.city">, </span>
+				<span ng-show="document.city">,</span>
 				{{document.state | lstring:locale}}
-				<br ng-show="document.city || document.state" /> {{document.postalCode | lstring:locale}}
-				<span ng-show="document.postalCode">, </span>
+				<br ng-show="document.city || document.state" />{{document.postalCode | lstring:locale}}
```
Comment (sbilodeau) [inline app/views/forms/view/view-contact.directive.html:83]: isn't better to use ng-bind from translation?
```suggestion
				<br ng-show="document.city || document.state" />
				<span ng-bing="document.postalCode | lstring:locale"></span>
```

**Review by sbilodeau — APPROVED** [review]

File: app/views/forms/view/view-contact.directive.html [hunk 2]
```diff
@@ -78,15 +78,16 @@ <h4>
 			<address class="km-value">
 				<div class="km-pre">{{document.address | lstring:locale}}</div>
 				{{document.city | lstring:locale}}
-				<span ng-show="document.city">, </span>
+				<span ng-show="document.city">,</span>
 				{{document.state | lstring:locale}}
```
Comment (blaisefonseca) [inline app/views/forms/view/view-contact.directive.html:82]: {{document.state | lstring:locale}}
```suggestion
				<span ng-bind="document.state | lstring:locale"></span>
```

File: app/views/forms/view/view-contact.directive.html [hunk 3]
```diff
@@ -78,15 +78,16 @@ <h4>
...
-				<br ng-show="document.city || document.state" /> {{document.postalCode | lstring:locale}}
-				<span ng-show="document.postalCode">, </span>
+				<br ng-show="document.city || document.state" />
+				<span ng-bing="document.postalCode | lstring:locale"></span>
+				<span ng-show="document.postalCode">,</span>
 				<b>{{document.country|term:locale}}</b>
```
Comment (blaisefonseca) [inline app/views/forms/view/view-contact.directive.html:86]: <b>{{document.country|term:locale}}</b>
```suggestion
				<b ng-bind="document.country|term:locale"></b>
```

File: app/views/forms/view/bch/view-risk-assessment.directive.html [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ﻿<div id="Record" class="record panel panel-default">
-	<div class="panel-heading" ng-class="::{'bg-orange':document.header.schema!='nationalRiskAssessment', 'bg-blue':document.header.schema=='nationalRiskAssessment'}">
+	<div class="panel-heading" ng-class="::{'bg-blue':document.header.schema!='nationalRiskAssessment', 'bg-orange':document.header.schema=='nationalRiskAssessment'}">
```
Comment (blaisefonseca) [inline app/views/forms/view/bch/view-risk-assessment.directive.html:2]: any reason the color is revered? nationa record are blue and reference reocrds are blue, can you just use `independentRiskAssessment` for orage and `nationalRiskAssessment` for blue

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**